front-end/src/components/Admin/TabbedArea/index.scss

```diff
@@ -212,4 +212,18 @@
       background-color: #fff;
     }
   }
+
```

Comment:
Do not push the same code changes into multiple branches. Make sure to start every new branch from master. It is not a big deal in this case, as the change is minor, but could be a problem with larger commits.

Reply:
Oh, sure. Forgot to switch to master